modules/network-observability-operator-install.adoc

```diff
@@ -17,7 +21,7 @@ You can install the Network Observability Operator using the {product-title} web
 +
 * *spec.agent.ebpf.Sampling* : Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the `FlowCollector` API reference, under spec.agent.ebpf. 
 * *spec.deploymentModel*: If you are using Kafka, verify Kafka is selected.
-* *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway.http-netobserv.svc:8080/api/logs/v1/network`
+* *Loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway.http.netobserv.svc:8080/api/logs/v1/network`
```

Comment:
Per Ronen's feedback. Also Steven Lee mentioned this before, I think I missed this one dash.

Reply:
In addition to this change, the dot (`.`) between `gateway` and `http` should be replaced by a dash (`-`)

Reply:
Addressed, just pushed the changes.